README.md

```diff
@@ -1,2 +1,190 @@
+
+**Note: for mock calls to these endpoints and example usage see test.js**
+
+### Attendify.js
+
+⚫ Current version of documentation for Attendify.js is in the `docs` folder and was generated by jsdoc. Latest version of docs is hosted [here](https://justanotherdevv.github.io/AttendifyXRPL/Attendify.html).
+
+### REST API endpoints
+
+This API is built using the ExpressJS framework and the XRPL library, it connect to the local Attendify.js class to perform most of the actions.
+
+⚫ `GET /api/mint` - Creates new claimable event and premints NFTs for it. You must have at least X XRP in order to use this to handle the reserve requirements for the NFTs and the minting process which uses [Tickets](https://xrpl.org/tickets.html), where X is amount of tokens that should be minted multiplied by 2 + additional XRP for feees.
+
+**@param {string} walletAddress** - Wallet address from user requesting claim
+
+**@param {integer} tokenCount** - Amount of NFTs that should be created for the event
+
+**@param {string} url** - Url to Image for event | Preferably this should be stored on IPFS
+
+**@param {string} title** - Title for event
+
+**@param {string} desc** - Description of event
+
+**@param {string} loc** - Location of event
+
+**@returns {object} result** - An object with details for the claim event that was successfully created
+
+**Structure of the example result object:**
+
+`{
+  eventId: 0,
+  account: 'rJ7t4Ei9Q2gs3uZoSLaVajE771Wt9GuuxR',
+  owner: 'rJnCJZZXSSnuDi9YVgrAatVqSvktTeXr5r',
+  URI: 'https://ipfs.io/ipfs/QmWveeBaaPQJhREN523qG5M3ciVmLA6Z271dA1qasVDixw',
+  title: 'test_title',
+  claimable: 3
+}
+     `
+
+Example request `http://localhost:4000/api/mint?walletAddress=rpHJL8bRW81shYm3LwwsXovABps6SM51RS&tokenCount=5&url=ipfs://QmQDDD1cNgnyhPC4pBLZKhVeu12oyfCAJoWr1Qc1QgbkPN&title=test_title&desc=test_description&loc=Warsaw`
+
+⚫ `GET /api/claim` - Checks if user is eligible for NFT claim or creates new offer that allows for claiming NFT for specific event. Endpoint returns response with sellOfferID that has to be accepted by user afterwards.
+
+**@param {string} walletAddress** - The wallet address of the user trying to claim
+
+**@param {integer} type** - The type of claim (1 for checking claim status and metadata, 2 for claiming)
+
+**@param {string} minter** - The minter address of the event
+
+**@param {string} eventId** - The ID of the event
+
+**@returns {object} result** - An object with the event metadata and offer for NFT
+
+**Possible responses**
+
+    -`404` - Event with specified ID does not exist
+
+    -`claimed` - The NFT for specified event was already claimed by this user
+
+    -`empty` - All NFTs for this particular event were already claimed
+
+    -`success` - If `onlyCheckStatus` parameter was true it indicated that current parameters for selected event were sent
+
+    -`transferred` - Indicates that new sell offer for NFT related to selected event was created successfully and details were sent to user
```

Comment:
Would it be possible to simplify the return type when it is successfully transferred to just show the NFTokenOffer the user gets instead of also showing the other NFTokens that are available? (It is a lot of information that the end user can't use, and is easy to get confused about which value to copy for the next step in the process)